skbio/util/_decorator.py

```diff
@@ -343,7 +342,6 @@ class classonlymethod(classmethod):
 
     def __get__(self, obj, cls=None):
         if obj is not None:
-            raise TypeError("Class-only method called on an instance. Use"
-                            " '%s.%s' instead."
-                            % (cls.__name__, self.__func__.__name__))
+            raise TypeError(f'Class-only method called on an instance. Use \
```

Comment:
Hi @mataton Glad that the extra whitespaces issue has been fixed in the instances being unit tested. However there are additional instances like this one, which are not unit tested (therefore the CI passed), but can still generate inconsistent screen output to the user. Can you fix these too? Thanks!

Reply:
@qiyunzhu Good catch, there were a couple of other instances where this error was made and I fixed those too.